root/layout.tt

```diff
@@ -109,7 +109,8 @@
                 <a href="//wiki.musicbrainz.org/" class="internal">[% l('Wiki') %]</a> |
                 <a href="https://community.metabrainz.org/" class="internal">[% l('Forums') %]</a> |
                 <a href="http://tickets.musicbrainz.org/" class="internal">[% l('Bug Tracker') %]</a> |
-                <a href="https://twitter.com/MusicBrainz" class="internal">[% l('Twitter') %]</a>
+                <a href="https://blog.musicbrainz.org/" class="internal">[% l('Blog') %]</a>
+                <a href="https://twitter.com/MusicBrainz" class="internal">[% l('Twitter') %]</a> |
```

Comment:
The `|` at the end needs to go to the line with “Blog”. Otherwise we have no separator between blog and Twitter, but a spurious one after the last entry.